eth_tester/backends/pyevm/main.py

```diff
@@ -471,6 +472,8 @@ def _get_normalized_and_unsigned_evm_transaction(self, transaction, block_number
         return evm_transaction
 
     def _get_normalized_and_signed_evm_transaction(self, transaction, block_number='latest'):
+        if transaction['from'] not in self._key_lookup:
+            raise ValidationError('"from" key not available, does this account exist?')
```

Comment:
This message seems like it could be improved a bit.

> No `"from"` key was provided in the transaction which is required for transaction signing

A bit more concise and hints at how the user can fix it.

Reply:
I am changing this message to `No valid "from" key was provided...` the reason is that one can specify a public key, but the the key isn't available to use (this is how I found the error, I specified a public key of a contract).

Reply:
If we wanted to get really fancy, we could list the valid keys and show the actual key, but that shouldn't hold up this PR.